operator/v1/types_etcd.go

```diff
@@ -43,6 +43,18 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// backendQuotaGiB sets the etcd backend storage size limit in gibibytes.
+	// The value should be an integer not less than 8 and not more than 32.
+	// When not specified, the default value is 8.
+	// +kubebuilder:default:=8
+	// +kubebuilder:validation:Minimum=8
+	// +kubebuilder:validation:Maximum=32
+	// +kubebuilder:validation:XValidation:rule="self>=oldSelf",message="etcd backendQuotaGiB may not be decreased"
+	// +openshift:enable:FeatureGate=EtcdBackendQuota
+	// +default:=8
```

Comment:
This still needs the colon removing

Reply:
Its not possibel .. verification fails when i remove it

Reply:
The verification should pass now you've added the omitempty back, is that not so?

Reply:
No it failed when i removed the `:`

Reply:
Please try again, I just checked your code out and I can run `make update` without it complaining, the previous issue was the lack of `omitempty` which is what alerted me to the fact it was missing

Reply:
ok i try now 👍🏽

Reply:
yes u r right, updated 👍🏽